docs/plan/20260416-pekko-phase-kernel-typed-ai-fix-step3.md

```
@@ -0,0 +1,15 @@
+# 実装計画
+
+## 対象
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)